host/sgx/sgxquote.c

```diff
@@ -63,58 +65,99 @@ static quote3_error_t (*_sgx_qe_get_quote)(
     uint32_t quote_size,
     uint8_t* p_quote);
 
+static quote3_error_t (*_sgx_qv_get_quote_supplemental_data_size)(
+    uint32_t* p_data_size);
+
+static quote3_error_t (*_sgx_qv_verify_quote)(
+    const uint8_t* p_quote,
+    uint32_t quote_size,
+    const struct _sgx_ql_qve_collateral_t* p_quote_collateral,
+    const time_t expiration_check_date,
+    uint32_t* p_collateral_expiration_status,
+    sgx_ql_qv_result_t* p_quote_verification_result,
+    sgx_ql_qe_report_info_t* p_qve_report_info,
+    uint32_t supplemental_data_size,
+    uint8_t* p_supplemental_data);
+
 #ifdef _WIN32
 
 #include <windows.h>
 
-#define LIBRARY_NAME "sgx_dcap_ql.dll"
+#define SGX_DCAP_QL_NAME "sgx_dcap_ql.dll"
+#define SGX_DCAP_QVL_NAME "sgx_dcap_quoteverify.dll"
+
 // Use LOAD_LIBRARY_SEARCH_SYSTEM32 flag since sgx_enclave_common.dll is part of
 // the Intel driver components and should only be loaded from there.
-#define LOAD_SGX_DCAP_QL() \
-    (void*)LoadLibraryEx(LIBRARY_NAME, NULL, LOAD_LIBRARY_SEARCH_SYSTEM32);
+#define LOAD_SGX_DCAP_LIB(libname) \
+    (void*)LoadLibraryEx((LPCSTR)libname, NULL, LOAD_LIBRARY_SEARCH_SYSTEM32);
 
-#define LOOKUP_FUNCTION(fcn) (void*)GetProcAddress((HANDLE)_module, fcn)
+#define LOOKUP_FUNCTION(module, fcn) (void*)GetProcAddress((HANDLE)module, fcn)
 
-#define UNLOAD_SGX_DCAP_QL() FreeLibrary((HANDLE)_module)
+#define UNLOAD_SGX_DCAP_LIB(module) FreeLibrary((HANDLE)module)
 
 #define SGX_DCAP_IN_PROCESS_QUOTING() \
     (GetEnvironmentVariableA("SGX_AESM_ADDR", NULL, 0) == 0)
 
+// In order to force CI to do QVL-based quote verification, just make
+// USE_SGX_ENV to TRUE temporarily, will convert it back before merge
+#define TRY_TO_USE_SGX_DCAP_QVL() TRUE
+//#define TRY_TO_USE_SGX_DCAP_QVL() \
+//    (GetEnvironmentVariableA("USE_SGX_QVL", NULL, 0) != 0)
```

Comment:
OK I saw you manually set USE_SGX_QVL be true so CI is actually using QVL verification right?
Recent CI failed and I saw it raised an error: multi-line comment for L104-105 here. You can just remove temporarily.